uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss

```diff
@@ -0,0 +1,36 @@
+.page-civicrm-leaveandabsences-dashboard {
+  h1.page-title,
+  #branding {
+    display: none !important;
+  }
+
+  #page {
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.admin-dashboard-page {
```

Comment:
1. `panel-heading` - Shoreditch has white color for heading background, here the color is different.
2. `dropdown-toggle`- Shoreditch does not have a button with transparent background and white font color.
3. `chr_admin-dashboard_side-nav  li.active`- Shoreditch does not have the variant of Tab/Nav that we need. The color of background for selected tab, is different here.

All the above are in respect to styleguide.